chat-widget/src/common/telemetry/TelemetryHelper.ts

```diff
@@ -274,4 +274,17 @@ export class TelemetryHelper {
         };
         BroadcastService.postMessage(telemetryEvent);
     }
+
+    public static logWebChatEvent = (logLevel: LogLevel, payload: TelemetryEventWrapper) => {
+        const telemetryEvent: ITelemetryEvent = {
+            eventName: TelemetryEvent.event,
+            logLevel: logLevel,
```

Comment:
should we default to debug in case log level is not defined?, what happens is payload is null?

Reply:
if payload is null, then for those respective fields within payload there would be empty data logged -- we'll still see telemetry for the fields that do have values